src/modules/Tab/Tab.js

```diff
@@ -114,7 +114,7 @@ class Tab extends Component {
     const { activeIndex } = this.state
 
     if (menu.tabular === true) {
-      menu.tabular = menuPosition === 'left' || 'right'
+      menu.tabular = menuPosition
     }
```

Comment:
this way we can specify `menu.tabular` as `true` if we just want to infer it from `menuPosition`
and as `left` or `right` if we want to specify ourselves